src/RoutesWrap.tsx

```diff
@@ -41,19 +41,7 @@ type CommonRoutesFeaturesProps = {
 };
 
 const afterChatContent = (loginFeature: boolean) => {
-  return (
-    <div className={s.afterChatContentWrap}>
-      <a
-        href="https://www.kontur.io/portfolio/disaster-ninja/"
-        target="_blank"
-        className={s.link}
-        rel="noreferrer"
-      >
-        <Text type="heading-m">{i18n.t('about')}</Text>
```

Comment:
if you remove translations - please also remove them from `src/core/localization/translations/en/common-messages.json`

Reply:
en and es both